scripts/prepare_sector_network.py

```diff
@@ -329,13 +330,21 @@ def add_hydrogen(n, costs):
         capital_cost=h2_capital_cost,
     )
 
-    if not snakemake.config["sector"]["hydrogen"]["network_routes"] == "greenfield":
+    # Hydrogen network:
+    # -----------------
+    if snakemake.config["sector"]["hydrogen"]["network"]:
         h2_links = pd.read_csv(snakemake.input.pipelines)
 
         # Order buses to detect equal pairs for bidirectional pipelines
         buses_ordered = h2_links.apply(lambda p: sorted([p.bus0, p.bus1]), axis=1)
 
-        if snakemake.config["clustering_options"]["alternative_clustering"]:
+        # Read the config.pypsa-earth.yaml file
+        yaml = ruamel.yaml.YAML()
+        file_path = "./config.pypsa-earth.yaml"
+        with open(file_path, "r") as file:
+            config_pypsa_earth = yaml.load(file)

```

Comment:
This can lead to problems unfortunately.
As an alternative, we could place an additional `configfile: "config.pypsa-earth.yaml"` at the beginning of the snakefile to merge the two yaml files before this point.
What do you think?
If we don't do that now, we'll need to do that before the merge

Reply:
I integrated the needed line from config.pypsa-earth.yaml into config.default.yaml in order to avoid using the above method.

Of course this can be nicely solved once we merge both models.